### ISSUE TITLE
[Doc]: Clarify the use for the "-Zone" parameter in New-AzVm

### ISSUE THREAD

**Issue by Matteo-T** [Azure/azure-powershell#19963]
### Description

Could you kindly clarify what the "-Zone" parameter in the New-AzVM cmdlet is about?

The documentation states what is kind of obvious ("Specifies the zone list of the virtual machine.") which is... not super helpful without the proper context. Also, the fact that it's just an array of strings does not help me figure out what values I should pass in.

So, my suggestion would be to elaborate a little on this parameter and, ideally, provide an example of use.

Linke to page: https://learn.microsoft.com/en-us/powershell/module/az.compute/new-azvm

Thanks!

**Comment by jaylabell** [comment]
HI @Matteo-T - Thank you for the insight. We will work on this request for an update.

**Comment by jaylabell** [comment]
The updates are currently pending.

**Comment by grizzlytheodore** [comment]
@Matteo-T, I've updated the documentation in the above Pull Request. 
It adds examples using the Zone parameter. It also has a reference to a page that explains what availability zones are in Azure.

One thing I did notice something. '-Zone' takes in an array of string (string[])  as input, but virtual machines currently do not support multiple availability zones. And currently, when a user provides more than one zone during VM creation, they will get a clear error message saying multiple availability zones is not supported for VMs. There are a few reasons for the input type being still an array:
- Zone is a property defined at the ARM level. And while some resources like a virtual machine only support one zone, resources like a virtual machine scale set supports multiple availability zones. We wanted to keep the zone parameter consistent across all resources, and they all take in an array of strings right now. 
- At the API level, a virtual machine takes in an array of strings for zones. Another reason for that is we may support multiple availability zones for VMs in the future. 
- If changing -Zone to just string input will be a breaking change for some customers and changing it back to array of strings when it is supported will be another breaking change.
